default.nix

```diff
@@ -36,6 +36,9 @@ let
     checks = recurseIntoAttrs {
       # `checks.tests` collect results of executing the tests:
       tests = collectChecks haskellPackages;
+
+      # checks that every tags in cabal.project exists on master branches:
```

Comment:
Using "tags" confused me initially. This is about commit hashes, not about git tags, but in `cabal.project`, `tag` is used for that purpose.

btw, "every tag" or "all tags .. exist"